packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,259 @@
+
+---
+
+## Overview
+
+A service-to-service app where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
+API service integrations access Okta APIs using the OAuth 2.0 [Client Credentials flow](/docs/concepts/oauth-openid/#client-credentials-flow), where access isn't associated with a user and you can restrict resources with scoped access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+
+> **Note:** Currently, Okta only supports the OAuth APIs listed in [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints).
+
+Customers can use any API service integration listed in the OIN catalog with their Okta tenant org. Each customer Okta org has its own authorization server that supports the Client Credentials flow. When a customer authorizes your API service integration to access their org, Okta generates a unique set of credentials (client ID and client secret) for that org.
+You must collect and [store these credentials](#save-customer-credentials) for each customer to allow your integration to work with your customer's Okta org.
+
+### API service integration Client Credentials flow
+
+At a high level, the OAuth 2.0 Client Credentials flow for an API service integration has the following steps:
+
+<div class="three-quarter">
+
+![Flowchart that displays the back and forth between the resource owner, authorization server, and resource server for API service integration's Client Credentials flow](/img/oin/api-service-creds-flow.png)
+
+</div>
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+skinparam monochrome true
+participant "Your app (customer tenant)" as client
+participant "Authorization Server (customer tenant Okta org)" as okta
+participant "Resource Server (customer tenant Okta APIs)" as app
+
+autonumber "<b>#."
+client -> okta: Access token request to /token
+okta -> client: Access token response
+client -> app: Request with access token
+app -> client: Response
+-->
+
+1. Your customer's service app instance makes an access token request to their Okta Authorization Server using their client credentials.
+
+   Your customer needs to install and authorize your integration in their Okta org so that Okta can accept the access token request. See [Set up an API service integration](https://help.okta.com/okta_help.htm?type=oie&id=csh-api-service-integrations) for customer tenant Admin Console instructions. After installing your integration, the customer obtains their client credentials and passes them to your app. The customer's app instance can now make an access token request to Okta. See [Request for access token](#request-an-access-token).
```

Comment:
```suggestion
   Your customer needs to install and authorize your integration in their Okta org so that Okta can accept the access token request. See [API Service Integrations](https://help.okta.com/okta_help.htm?type=oie&id=csh-api-service-integrations) for customer tenant Admin Console instructions. After installing your integration, the customer obtains their client credentials and passes them to your app. The customer's app instance can now make an access token request to Okta. See [Request for access token](#request-an-access-token).
```
Product docs topic name change